ENIGMAsystem/SHELL/Platforms/xlib/XLIBwindow.cpp

```diff
@@ -632,15 +632,6 @@ void initkeymap() {
 
 namespace enigma_user {
 
-void io_handle() {
-  enigma::input_push();
-  while (XQLength(disp)) {
-    DEBUG_MESSAGE("processing an event...", MESSAGE_TYPE::M_INFO);
-    if (enigma::handleEvents() > 0) exit(0);
```

Comment:
X11 was already calling the full thing @JoshDreamland

Reply:
Also there really is no such thing as full or partial event loops, TranslateMessage always calls the same window procedure. All platforms are the same in this regard.

Reply:
Does handleEvents already loop until the queue is completely empty? If that's the case, this change should be fine.

Reply:
Yes @JoshDreamland handleEvents draining the queue that be the case on all platforms and xlib.